Sources/Callout/CalloutSize.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+extension Warp {
+    public enum CalloutSize {
+        case small
+        case `default`
+
+        public var typography: Warp.Typography {
+            switch self {
+            case .small:
+                Typography.caption
+            case .default:
+                Typography.body
+            }
+        }
+
+        public var cornerRadius: Double {
+            switch self {
+            case .small:
+                6
+            case .default:
+                8
+            }
+        }
+
+        public var verticalPadding: Double {
```

Comment:
I think you are not using this anywhere. That's why the height looks a little bit smaller than design.
![Simulator Screenshot - iPhone 15 - 2024-02-07 at 18 27 06](https://github.com/warp-ds/warp-ios/assets/89784012/dba17745-bedf-4439-816f-323613e86ff7)
<img width="441" alt="Screenshot 2024-02-07 at 18 27 34" src="https://github.com/warp-ds/warp-ios/assets/89784012/27a8256d-bb4f-4959-9005-6895c3fc642e">

Reply:
At least the ones with vertical arrows, the leading and trailing seems correct.